test/crypto/rsa_sign_test.dart

```diff
@@ -38,48 +37,530 @@ void main() {
     // 000000e0  31 0d 3c 36 6c bf 14 df  dc 2d 53 c7 ca d1 ec 6d
     // 000000f0  95 37 2f 86 14 da 6c 04  a1 fd 45 fa 95 e0 04 bf
     test('encrypt-hello-world', () {
-      expect(signer.sign(ASCII.encode('hello world')), equals(
-          [89, 157, 111, 129, 193, 15, 214, 241, 88, 70, 45, 77, 201, 184, 105,
-           29, 177, 224, 224, 38, 164, 222, 73, 216, 79, 90, 172, 219, 129, 171,
-           16, 39, 58, 244, 90, 248, 187, 218, 169, 132, 190, 199, 90, 251, 185,
-           46, 10, 102, 143, 120, 213, 203, 201, 130, 11, 87, 54, 252, 188, 66,
-           27, 245, 250, 118, 183, 1, 76, 188, 45, 185, 254, 32, 85, 98, 245,
-           135, 140, 188, 227, 88, 166, 198, 138, 239, 22, 200, 74, 133, 1, 110,
-           223, 5, 67, 200, 239, 53, 55, 159, 27, 41, 87, 235, 199, 147, 137,
-           117, 245, 101, 129, 10, 108, 140, 68, 53, 173, 115, 137, 144, 83, 66,
-           38, 243, 49, 169, 6, 241, 50, 32, 72, 163, 225, 104, 61, 134, 103,
-           69, 116, 25, 145, 117, 201, 40, 202, 139, 51, 99, 237, 162, 177, 144,
-           230, 225, 10, 31, 135, 236, 2, 248, 146, 3, 207, 14, 48, 73, 176,
-           241, 114, 41, 163, 156, 46, 204, 124, 135, 101, 17, 31, 56, 52, 211,
-           62, 254, 175, 142, 49, 240, 16, 31, 245, 113, 221, 144, 246, 199,
-           186, 93, 16, 12, 99, 235, 164, 60, 165, 23, 154, 153, 82, 45, 182,
-           39, 150, 140, 226, 68, 99, 53, 31, 4, 111, 184, 49, 230, 212, 71,
-           49, 13, 60, 54, 108, 191, 20, 223, 220, 45, 83, 199, 202, 209, 236,
-           109, 149, 55, 47, 134, 20, 218, 108, 4, 161, 253, 69, 250, 149, 224,
-           4, 191]));
+      expect(
+          signer.sign(ASCII.encode('hello world')),
+          equals([
+            89,
+            157,
+            111,
+            129,
+            193,
+            15,
+            214,
+            241,
+            88,
+            70,
+            45,
+            77,
+            201,
+            184,
+            105,
+            29,
+            177,
+            224,
+            224,
+            38,
+            164,
+            222,
+            73,
+            216,
+            79,
+            90,
+            172,
+            219,
+            129,
+            171,
```

Comment:
PTAL – dartfmt has rules around this. If you add a line comment, it will not try to expand things out